packages/mesos/extra/dcos-mesos-slave-public.service

```diff
@@ -21,4 +21,4 @@ ExecStartPre=/bin/ping -c1 leader.mesos
 ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-mesos-slave-public
 ExecStartPre=/opt/mesosphere/bin/make_disk_resources.py /var/lib/dcos/mesos-resources
 ExecStartPre=/bin/bash -c 'for i in /proc/sys/net/ipv4/conf/*/rp_filter; do echo 2 > $i; echo -n "$i: "; cat $i; done'
-ExecStart=$PKG_PATH/bin/mesos-slave
+ExecStart=/bin/bash -c '$PKG_PATH/bin/mesos-slave 2>&1 | tee -a /var/log/mesos/mesos-agent.log'
```

Comment:
using bin bash like this isn't allowed in our sytemd units. It makes systemd have a harder time tracking things,  and is overly complicated for an exec start line. See: https://github.com/dcos/dcos/blob/master/docs/systemd-rules.md